### PR TITLE
[CONS CONTENT] CHANGELOG updates for 2023-04-11 and following

### LINKED ISSUE #716: [BUG] - Broken link on quansight.com/consulting
### What site is this for?

Quansight Consulting

### Expected behavior

On https://quansight.com/consulting, the "Learn more" button under "Infrastructure, Scaling & Acceleration" should point to: https://quansight.com/infrastructure-scaling-acceleration
 

### Actual behavior

Currently, it points to an outdated link: https://quansight.com/service-infrastructure-scaling-acceleration

### How to Reproduce the problem?

Screenshot 2023-04-07 at 8 26 42 PM

### Anything else?

We can also consider adding a redirect from the old link `https://quansight.com/service-infrastructure-scaling-acceleration` to the new slug https://quansight.com/infrastructure-scaling-acceleration

cc @bskinn

### PR DESCRIPTION
**Pull request by bskinn**
- Storyblok changes to close #716.

  Something appears to have gotten corrupted in Storyblok's database for that  Board List Item component. Recreating the component and deleting the old one appears to have fixed the problem.

- Update @Abaal title to 'COO'

- Revise Consulting site header nav as discussed, and add link to new Nebari Services page (tandem change with #709)

- Revise Consulting footer nav to include a link to the new Nebari services page

- Change all mentions of QHub to Nebari, on Jupyter Technologies services page
  - Title and icon of tile
    
    image

  - Logos in open source projects logo spread

    image

### CODE REVIEW

**Comment by bskinn** [comment]
@gabalafou I believe all of the SB changes lined up for this build to live are complete and reflected herein.

This PR is ready for your review. Please proceed to merge after you review/approve if all is in order.

**Review by gabalafou — APPROVED** [review]

**Pull request merged**
